src/components/application_manager/test/commands/mobile/delete_file_test.cc

```diff
@@ -197,6 +197,28 @@ TEST_F(DeleteFileRequestTest, Run_InvalidFile_UNSUCCESS) {
   command_->Run();
 }
 
+TEST_F(DeleteFileRequestTest, Run_ValidFileNameWithGenericError_SUCCESS) {
+  const std::string kFileName = "read_only.txt";
+  (*message_)[am::strings::msg_params][am::strings::sync_file_name] = kFileName;
+
+  EXPECT_CALL(mock_app_manager_, application(_)).WillOnce(Return(app_));
```

Comment:
@okozlovlux Actually in almost all test cases this arg is skipped so I tought it is done on purpose, but I will fill the arguement, maybe you're right and it's an important check.